### PR TITLE
New jenkinsfile for deploying plugin site via new kubnernets infrastructure

### PR DESCRIPTION
**Pull request by halkeye**

### CODE REVIEW

File: PodTemplates.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+apiVersion: "v1"
+kind: "Pod"
+metadata:
+  labels:
+    jenkins: "agent"
+    job: "plugin-site"
+spec:
+  tolerations:
+    - key: "os"
```
Comment (halkeye) [inline PodTemplates.yaml:9]: @olblak i removed highmem from the spec

do i need job and the os selector and stuff?

**Review by timja — APPROVED** [review]

**Review by oleg-nenashev — APPROVED** [review]
Looks fine.

**Comment by halkeye** [comment]
I guess i could just merge it, its not being used yet, but I was kinda curious about the question i left open so waiting for olblak to come back

File: PodTemplates.yaml (hunk 1 above)
Comment (timja) [reply PodTemplates.yaml:9]: seems like you have the OS one which looks good

**Review by timja — APPROVED** [review]

**Pull request merged**